src/main/webapp/app/exercises/programming/shared/instructions-render/step-wizard/programming-exercise-instruction-step-wizard.scss

```diff
@@ -4,12 +4,6 @@
     border-bottom: 1px solid var(--programming-exercise-instruction-step-wizard-card-header-border);
 }
 
-.stepwizard-row {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-}
-
```

Comment:
This was only moved down to better reflect the hierarchy/order in the html template